MapboxAndroidDemo/build.gradle

```diff
@@ -111,7 +114,8 @@ dependencies {
     implementation dependenciesList.supportConstraintLayout
 
     // Mapbox dependencies
-    api(dependenciesList.mapboxMapSdk)
+    nogpservicesImplementation dependenciesList.mapboxChinaPlugin
+    gpservicesApi(dependenciesList.mapboxMapSdk)
```

Comment:
Considering that the Google Play Services-free flavor is now only targeting the China use case (as opposed to non-Play Services stores in general), how about taking this opportunity to rename the two flavors two something clearer, like `china` instead of `nogpservices`? Also, one of the entries include the brackets, while the other doesn't.

Reply:
Fixed